style.css

```diff
@@ -63,6 +95,229 @@ h1 {
   border: 2px solid #dfe1e6;
 }
 
+#work-section h2 {
+  font-size: 40px;
+  text-align: center;
+  color: #172b4d;
+  padding: 30px;
+}
+
+#card-container {
+  width: 100%;
+  display: grid;
+  justify-content: center;
+  padding: 10px;
+  gap: 15px;
+  grid-template-columns: repeat(auto-fill, 327px);
+}
+
+#about-me-text {
+  background-image: url('./images/about-me-mobile.png');
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px 0 20px 0;
+  gap: 20px;
+  background-size: cover;
+  background-position: 100%;
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+#about-me-text h2 {
+  font-size: 40px;
+  font-weight: bold;
+  color: #172b4d;
+}
+
+#about-me-text p {
```

Comment:
- [ ] Kindly adhere to the project requirement because as I can see you are not using the Correct `font-family`, `font-color` and etc. which makes your design `inconsistent` and looks unprofessional. Please have a look at the below-attached image and table.

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-05-04 15-03-39](https://user-images.githubusercontent.com/66476329/236190947-735236ee-ca90-4167-8455-c3c87177899e.png)

</td>
<td>

![Screenshot from 2023-05-04 15-03-53](https://user-images.githubusercontent.com/66476329/236190830-237d4d1d-c04a-4051-864e-47141e9e9e32.png)

</td>
</tr>
</table>